samples/highcharts/website/homepage-2021/demo.js

```diff
@@ -888,7 +888,15 @@ const sankey = {
                         } else {
                             clearInterval(sankey);
                         }
-                    }, 10);
+                    }, 10);*/
+
+                    for (let ii = 0; ii < sankeyData.length; ++ii) {
+                        chart.series[0].addPoint({
```

Comment:
When in a loop, `addPoint` should be called with the `redraw` parameter set to false, and `chart.redraw()` be called after. Otherwise, the chart will waste browser resources on redrawing for every iteration.